src/Workspaces/VisualBasic/Portable/Editing/VisualBasicImportAdder.vb

```diff
@@ -0,0 +1,90 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Collections.Immutable
+Imports System.Composition
+Imports System.Threading
+Imports Microsoft.CodeAnalysis
+Imports Microsoft.CodeAnalysis.Editing
+Imports Microsoft.CodeAnalysis.Host.Mef
+Imports Microsoft.CodeAnalysis.Options
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Microsoft.CodeAnalysis.VisualBasic.Utilities
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Editing
+    <ExportLanguageService(GetType(ImportAdderService), LanguageNames.VisualBasic), [Shared]>
+    Partial Friend Class VisualBasicImportAdder
+        Inherits ImportAdderService
+
+        Protected Overrides Sub GetExistingImportedNamespaces(document As Document, model As SemanticModel, namespaces As HashSet(Of INamespaceSymbol))
+            namespaces.AddRange(model.Compilation.MemberImports.OfType(Of INamespaceSymbol))
+
+            ' consider all imports clauses
+            Dim root = DirectCast(model.SyntaxTree.GetRoot(), CompilationUnitSyntax)
+            For Each import In root.Imports
```

Comment:
Declaring a loop variable without `As` clause may result in reusing existing field or property from an outer scope (which might be added later).